src/datastore.js

```diff
@@ -77,19 +77,35 @@ class DataStore extends EventEmitter {
 		return _.find(this._data, filter);
 	}
 
-	update(data) {
+	update(data, emitEvents) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method 'update' has a complexity of 5. (complexity)](https://app.codacy.com/app/vjfalk/AwesomeCartJS/pullRequest?prid=2579185)